config/database.yml

```diff
@@ -1,6 +1,6 @@
 default: &default
   adapter: postgresql
-  pool: <%= ENV["DB_POOL"] || ENV['MAX_THREADS'] || 5 %>
+  pool: <%= ENV["DB_POOL"] || (if Sidekiq.server? then Sidekiq[:concurrency] else ENV['MAX_THREADS'] end) || 5 %>
```

Comment:
```suggestion
  pool: <%= ENV["DB_POOL"] || (Sidekiq.server? ? Sidekiq[:concurrency] : ENV['MAX_THREADS']) || 5 %>
```

Reply:
@ClearlyClaire I like it, and I tried to use ternary operator (?:), but Prettier YML Lint failed.... 
(Due to force push, history is gone... I'm sorry about that)
<img width="893" alt="image" src="https://github.com/mastodon/mastodon/assets/1138433/fba7a872-1c18-4b1e-b01d-3944c585726f">

Reply:
Oh, I suppose prettier does not handle ERB well.

cc @nschonni

Reply:
@ClearlyClaire Should I changes to ternary operator with disabling prettier this file? Or keeping this PR?

Reply:
I think `if` `then` `else` is fine, if not ideal. I'd still like another pair of eyes for the whole feature, though.